package.json

```diff
@@ -20,6 +20,8 @@
     "@fusebit/oauth-connector": "1.3.1",
     "@js-joda/core": "3.2.0",
     "@js-joda/timezone": "2.5.0",
+    "@types/express": "^4.17.13",
+    "@types/node-fetch": "^2.6.1",
```

Comment:
We export types from these packages.  Not including them under `dependencies` leads to errors like this:
```
node_modules/@hyperproof/hypersync-sdk/lib/common/ApiClient.d.ts:2:39 - error TS7016: Could not find a declaration file for module 'node-fetch'. '/home/andrew/hp/sdk-samples/hypersync-mysql/node_modules/@hyperproof/hypersync-sdk/node_modules/node-fetch/lib/index.js' implicitly has an 'any' type.
```

See: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html